src/main/kotlin/apply/ui/api/ApplicantRestController.kt

```diff
@@ -55,6 +55,16 @@ class ApplicantRestController(
         return ResponseEntity.noContent().build()
     }
 
+    @PostMapping("/authentication-code")
+    fun generateAuthenticationCode(
+        @RequestParam email: String
+    ): ResponseEntity<Unit> {
+        val authenticateCode = applicantAuthenticationService
+            .generateAuthenticationCode(email)
+        mailService.sendAuthenticationCodeMail(email, authenticateCode)
+        return ResponseEntity.status(HttpStatus.NO_CONTENT).build()
```

Comment:
```suggestion
        return ResponseEntity.noContent().build()
```
위에랑 동일하게 맞춰주면 좋을 것같아요!

Reply:
반영해주었습니다~